suite/e2e/support/pageObjects/trading/tradingPage.ts

```diff
@@ -524,6 +566,9 @@ export class TradingPage {
 
     @step()
     async openConfirmAndSendModal() {
+        await expect(this.confirmOnTrezorAndSend).toBeVisible({ timeout: 30_000 });
+        await expect(this.confirmOnTrezorAndSend).toBeEnabled();
+
         await this.confirmOnTrezorAndSend.click();
```

Comment:
click action in playwright asserts both visibility and if the element is enabled. So I would suggest to instead just add timeout on click and save code lines.
```suggestion
        await this.confirmOnTrezorAndSend.click({ timeout: 30_000 });
```

Reply:
Good point, thanks. Fixed ✔️